src/server/login/controller.js

```diff
@@ -8,6 +8,8 @@ const loginController = {
   options: {
     auth: 'defra-id'
   },
+  /* @fixme: code coverage */
+  /* v8 ignore next */
```

Comment:
Discussed with @loveboat, this requires a different approach to testing which can be dealt with at a later stage